api_server.go

```diff
@@ -64,23 +70,23 @@ func logRequest(handlerFunc APIHandlerFunc, r *http.Request) ([]byte, int) {
 		"request_method":  r.Method,
 		"request_size":    r.ContentLength,
 		"request_seconds": time.Since(start).Seconds(),
-		"status_code":     code,
+		"status_code":     results.code,
 	}
 	switch true {
-	case code >= 500:
+	case results.code >= 500:
 		logger.WithFields(fields).Error("request failed")
-	case 400 <= code && code < 500:
+	case 400 <= results.code && results.code < 500:
```

Comment:
```suggestion
	case results.code >= 400:
```
Slightly more readable.